types/fork_choice.yaml

```diff
@@ -0,0 +1,35 @@
+Node:
+  type: object
+  description: "fork choice node attributes"
+  properties:
+    slot:
+      allOf:
+        - $ref: './primitive.yaml#/Uint64'
+        - description: "The slot to which this block corresponds."
+    block_root:
+      allOf:
+        - $ref: './primitive.yaml#/Root'
+        - description: "The signing merkle root of the `BeaconBlock`."
+    parent_root:
+      allOf:
+        - $ref: './primitive.yaml#/Root'
+        - description: "The signing merkle root of the parent `BeaconBlock`."
+    justified_epoch:
+      $ref: './primitive.yaml#/Uint64'
+    finalized_epoch:
+      $ref: './primitive.yaml#/Uint64'
+    weight:
+      $ref: './primitive.yaml#/Uint64'
+    validity:
+      type: string
+      enum: [VALID, INVALID, OPTIMISTIC]
+    execution_payload_root:
+      allOf:
+        - $ref: './primitive.yaml#/Root'
+        - description: "The tree hash merkle root of the `execution_payload` for the `BeaconBlock`."
```

Comment:
Lighthouse doesn't keep this on-hand in fork choice. Instead we store the `execution_payload.block_hash`.

This is a bit unfortunate, but if that's how our fork choice impls diverge maybe the `execution_payload_root` and `execution_block_hash` should go into `extra_data` as optional extras?

Reply:
it should be the block hash, this is a mistake. I'd rename it `execution_block_hash`.